pom.xml

```diff
@@ -57,6 +57,12 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>ionicons-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>display-url-api</artifactId>
+      <!-- TODO: Remove once included in BOM -->
+      <version>2.3.9</version>
```

Comment:
Without this, the changes in `PipelineGraphDisplayURLProvider` to move away from extending `ClassicDisplayURLProvider` directly would result in infinite recursion in some cases.